third_party/bigframes_vendored/constants.py

```diff
@@ -26,4 +29,5 @@
     "Abstract method. You have likely encountered a bug. "
     "Please share this stacktrace and how you reached it with the BigQuery DataFrames team. "
     f"{FEEDBACK_LINK}"
+    f"You are currently running BigFrames version {BF_VERSION}"
```

Comment:
Please put this information in the feedback link message, not just the abstract error message. The feedback link gets used in a lot more contexts than this message.